pkg/scalers/arangodb_scaler.go

```diff
@@ -31,37 +30,37 @@ type dbResult struct {
 
 // arangoDBMetadata specify arangoDB scaler params.
 type arangoDBMetadata struct {
-	// Specify arangoDB server endpoint URL or comma separated URL endpoints of all the coordinators.
+	// Specify arangoDB server endpoint URL or comma separated URL Endpoints of all the coordinators.
 	// +required
-	endpoints string
+	Endpoints string `keda:"name=endpoints, order=authParams;triggerMetadata"`
 	// Authentication parameters for connecting to the database
 	// +required
-	arangoDBAuth *authentication.AuthMeta
+	arangoDBAuth *authentication.AuthMeta // TODO: Fix this
 	// Specify the unique arangoDB server ID. Only required if bearer JWT is being used.
 	// +optional
 	serverID string
 
 	// The name of the database to be queried.
 	// +required
 	dbName string
-	// The name of the collection to be queried.
+	// The name of the Collection to be queried.
 	// +required
-	collection string
-	// The arangoDB query to be executed.
+	Collection string `keda:"name=collection, order=triggerMetadata"`
+	// The arangoDB Query to be executed.
 	// +required
-	query string
+	Query string `keda:"name=query, order=triggerMetadata"`
 	// A threshold that is used as targetAverageValue in HPA.
 	// +required
-	queryValue float64
+	QueryValue float64 `keda:"name=queryValue, order=triggerMetadata ,default=0"`
```

Comment:
micro nit: technically it doesn't matter but space after the comma satisfies my OCD :)
```suggestion
	QueryValue float64 `keda:"name=queryValue, order=triggerMetadata, default=0"`
```

Reply:
No problem at all. Will make the changes